src/app/components/Home/Office/Canvas/PopUpInfoPlace/index.tsx

```diff
@@ -146,45 +132,53 @@ const PopUpInfoPlace: React.FC<IPopUpInfoPlace> = ({
 
   const [dataSelectedWorkplace, setDataSelectedWorkplace] =
     useState<DataSelectedWorkplace>({
-      ...dataAboutWorkplace,
-      idOffice: selectedOffice.id,
-      selectFloor: `${selesctedFloor - 1}`,
-      occupant: user,
-      isDisabled: false,
+      idOffice: '',
+      selectFloor: '',
+      blocked: false,
+      endReservation: '',
+      label: '',
+      occupant: '',
+      palceIndex: '',
+      startReservation: '',
+      type: '',
     });
 
+  const { occupant } = dataAboutWorkplace ?? {};
+  const isDisabled = !!useMemo(
```

Comment:
better to update function inside `useMemo` to return `boolean` (for example `Boolean(occupant && user !== occupant)`)